test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Variable.Tests.ps1

```diff
@@ -145,7 +145,8 @@ Describe "Get-Variable" -Tags "CI" {
 	It "Should not be able to clear a global scope variable using the local switch" {
 	    New-Variable globalVar -Value 1 -Scope global -Force
 
-	    Get-Variable -Name globalVar -Scope local -ErrorAction SilentlyContinue | Should Throw
+	    Get-Variable -Name globalVar -Scope local -ErrorAction SilentlyContinue -ErrorVariable removeGlobalAsLocal
+        $removeGlobalAsLocal.FullyQualifiedErrorId | Should Be "VariableNotFound,Microsoft.PowerShell.Commands.GetVariableCommand"
```

Comment:
Is this line indented correctly?

Reply:
there is a sad combination of tabs and spaces in this file, I'll fix up this line and save the whitespace fixing for a later PR